pkg/stanza/tokenize/multiline_test.go

```diff
@@ -47,11 +67,28 @@ func TestLineStartSplitFunc(t *testing.T) {
 				"LOGSTART 234 log2",
 			},
 		},
+		{
+			Name:                  "TwoLogsLineStartOmitPattern",
+			Pattern:               `^LOGSTART \d+ `,
+			OmitPatternFromRecord: true,
+			Input:                 []byte("LOGSTART 123 LOGSTART 345 log1\nLOGSTART 234 log2\nLOGSTART 345 foo"),
+			ExpectedTokens: []string{
+				"LOGSTART 345 log1",
+				"log2",
+			},
+		},
```

Comment:
It would be nice to see this case without the `^` as well. I think it would output an empty token:
```go
ExpectedTokens: []string{
	"",
	"log1",
	"log2",
},
```